spec/controllers/barcode_items_controller_spec.rb

```diff
@@ -60,9 +60,10 @@
       end
     end
 
-
     describe "DELETE #destroy" do
-      subject { delete :destroy, params: default_params.merge({ id: create(:barcode_item) }) }
+      pending "it doesn't allow a user to delete someone else's barcode"
+      pending "it doesn't allow a non-admin to delete a global barcode"
```

Comment:
I think this highlights somethign that is probably going to be an issue in the future and will likely warrant a discussion at some point -- A deeper concept of roles/access rights.

Reply:
YES. exactly.

It felt out of scope for this PR but since it crossed my mind and seemed relevant I thought I'd at least put a hook there for later